src/test/java/yevhenii/lostfilmdemo/services/impl/TVSeriesServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import org.jooq.Record;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import yevhenii.lostfilmdemo.convertors.TVRecordConvertor;
+import yevhenii.lostfilmdemo.entity.TVSeries;
+import yevhenii.lostfilmdemo.jooq.generated.tables.records.TvSeriesRecord;
+import yevhenii.lostfilmdemo.kafka.TVSeriesSender;
+import yevhenii.lostfilmdemo.repository.impl.TVSeriesRepositoryImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class TVSeriesServiceImplTest {
+    @Mock
+    TVSeriesRepositoryImpl repository;
+    @InjectMocks
+    TVSeriesServiceImpl service;
+    @Mock
+    TVRecordConvertor convertor;
+    @Mock
+    TVSeriesSender sender;
+
+    private static TVSeries series;
+
+    @BeforeAll
+    static void init() {
+
+        series = TVSeries.builder()
+                .lastUpdate("update").episode(1)
+                .image("image").russianName("rname")
+                .name("name").season(2)
+                .link("link").build();
+    }
+
+    @Test
+    void save() {
+
+        doNothing().when(repository).create(isA(Record.class));
+        doReturn(Optional.empty()).when(repository).read(isA(String.class));
+        doReturn(new TvSeriesRecord()).when(convertor).createRecord(any(TVSeries.class));
+        doNothing().when(sender).send(any(), isA(TVSeries.class));
+
+        service.save(series);
+        verify(repository).create(any(TvSeriesRecord.class));
```

Comment:
don't use any, it leads to bugs. You mocked TvSeriesRecord from converter at 52 line, so verify that that object was passed into repo